tools/RAiDER/cli/raider.py

```diff
@@ -97,7 +97,7 @@ def read_template_file(fname):
                 raise ValueError(f"Unknown look direction {value}")
             template['look_dir'] = value.lower()
         if key == 'cube_spacing_in_m':
-            template[key] = float(value)
+            template[key] = float(value) if isinstance(value, str) else value
```

Comment:
Is this needed? There is also the case of passing `int`s etc. If a general object is passed you don't necessarily want to allow the code to proceed.